spec/poros/food_poro_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Food do
+  describe 'initialize' do
+    it 'creates a food item' do
+      actual = Food.new({
+        gtin_upc: '123456',
+        description: 'num',
+        brand_owner: 'me',
+        ingredients: 'toes'
+      })
+
+      expect(actual.gtin_upc).to eq('123456')
+      expect(actual.description).to eq('num')
+      expect(actual.brand_owner).to eq('me')
+      expect(actual.ingredients).to eq('toes')
+    end
+  end
+end
```

Comment:
Instead of creating a food with your own values you come up with, grab a json response that you'd send to this initialize method, and make sure that the food poro is able to grab the correct keys/values from the json that's passed, and successfully create teh object you're expecting.